docs/rnode-api/rnode-openapi.json

```diff
@@ -38,9 +38,13 @@
           "description": "WebApi.scala",
           "url": "https://github.com/rchain/rchain/blob/dev/node/src/main/scala/coop/rchain/node/api/WebApi.scala"
         },
-        "tags": ["blocks"],
+        "tags": [
+          "blocks"
+        ],
         "operationId": "getBlocks",
-        "produces": ["application/json"],
+        "produces": [
+          "application/json"
+        ],
```

Comment:
Why the whitespace changes? The previous style was from a vs-code plug-in (I suppose documenting that only in a commit message wasn't very thorough).

If you really want to keep them, could you put them in their own commit?

Reply:
Sorry for the confusion

Reply:
Actually what was the formatter you used?

Reply:
The formatter was part of https://github.com/42Crunch/vscode-openapi